src/scalar.rs

```diff
@@ -1297,6 +1252,20 @@ mod test {
         ],
     };
 
+    /// The largest valid scalar (not mod l). Remember for NAF computations, the top bit has to be
+    // 0. So the largest integer a scalar can hold is 2^255 - 1. Addition and subtraction are
+    // broken on unreduced scalars. The only thing you can do with this is multiplying with a curve
+    // point (and actually also scalar-scalar multiplication, but that's just a quirk of our
+    // implementation).
+    #[cfg(feature = "precomputed-tables")]
+    static LARGEST_UNREDUCED_SCALAR: Scalar = Scalar {
```

Comment:
I guess this is just using it for tests? But regardless it seems like it could be defined in a single place

Reply:
Agreed. None of the tests seem to wanna copy values around, but I can refactor so `edwards.rs` just pulls in `scalar::test::*`